index.js

```diff
@@ -1,17 +1,25 @@
 const express = require('express');
+const session = require('express-session');
 const http = require('http');
 const Memcached = require('memcached');
 const admin = require('firebase-admin');
 const serviceAccount = require('./keys/firestore-key.json');
 const bodyParser = require('body-parser');
 const dateFormat = require('date-format');
+const randLoc = require('random-location');
+const fs = require('fs');
+const e = require('express');
```

Comment:
We `require('express')` twice in this file, once on line one and once on line 11. Should be an easy fix, just change line twelve to `const app = e();` or change all references to `e()` to `express()`